src/utils/validators.js

```diff
@@ -14,36 +12,20 @@ import i18n from 'src/utils/i18n/i18n';
 export const isNumeric = (value) => /^(-?[0-9]+\.?[0-9]*|\.[0-9]+)$/.test(value);
 
 /**
- * Validates the given address with respect to the tokenType
- * @param {String} tokenType
+ * Validates the given address
+ *
  * @param {String} address
- * @param {Object} network The network config from Redux store
  * @returns {Number} -> 0: valid, 1: invalid, -1: empty
  */
-export const validateAddress = (tokenType, address, network) => {
+export const validateAddress = (address) => {
   if (address === '') {
     return -1;
   }
 
-  switch (tokenType) {
-    // Reference: https://github.com/bitcoinjs/bitcoinjs-lib/issues/890
-    case tokenMap.BTC.key:
-      try {
-        bitcoin.address.fromBase58Check(address); // eliminates segwit addresses
-        bitcoin.address.toOutputScript(address, network.networks.BTC.network);
-        return 0;
-      } catch (e) {
-        return 1;
-      }
-
-    case tokenMap.LSK.key:
-      try {
-        return cryptography.validateBase32Address(address) ? 0 : 1;
-      } catch (e) {
-        return 1;
-      }
-    default:
-      return 1;
+  try {
+    return cryptography.validateBase32Address(address) ? 0 : 1;
+  } catch (e) {
+    return 1;
```

Comment:
is it possible to return boolean? 
then we can remove unnecessary condition like this `validateAddress(summary.address) === 0`

Reply:
Number is usually preferred over boolean. Because it can return the number of issues discovered. This is the approach we have taken in several places throughout the app. I'd rather keep it.